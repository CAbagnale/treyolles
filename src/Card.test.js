import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';

describe('Card component tests', () => {

	it('renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(<Card />, div);
		ReactDOM.unmountComponentAtNode(div);
	});

	it('renders the UI as expected with title = myTitle and content = myContent', () => {
		const tree = renderer
			.create(
				<div name="Card">
					<button type="button">delete</button>
					<h3>title={'testTitle'}</h3>
					<p>content={'testContent'}</p>
				</div>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
    });
    
});