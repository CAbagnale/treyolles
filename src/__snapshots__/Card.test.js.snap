// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component tests renders the UI as expected with title = myTitle and content = myContent 1`] = `
<div
  name="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    title=
    testTitle
  </h3>
  <p>
    content=
    testContent
  </p>
</div>
`;
